lambda/create-config.ts

```diff
@@ -0,0 +1,83 @@
+import { Handler, APIEventResponse } from './handler';
+import * as AWS from 'aws-sdk';
+import * as util from './util';
+
+const s3 = new AWS.S3();
+
+/**
+ * Creates config.js at the root of the S3 bucket.
+ * 
+ * We do this with a custom resource because the web app needs deploy-time values.
+ */
+class CreateConfigHandler {
+
+    /**
+     * The event handler.
+     */
+    public async handle(event: any): Promise<any> {
```

Comment:
Nit: we use 2 space indents.

Reply:
Right. Very hard to get used to. Disagree and commit!